dashboard/app/views/scripts/show.html.haml

```diff
@@ -70,11 +70,22 @@
           - @script.courses.each do |course|
             %li= link_to course_path(course)
       - levels = Script.get_from_cache(@script.name).stages.map{ |stage| stage.script_levels.map{ |sl| sl.level }}.flatten
+      .row
+        .span1
+          = "Index"
+        .span3
+          = "Level name"
+        .span3
+          = "Template level"
+        .span3
+          = "Contained levels"
       - levels.each_with_index do |level, index|
         .row
           .span1
             = index + 1
           .span3
             = level.name
-          .span8
-            = level.instructions
+          .span3
+            = level.properties['project_template_level_name']
+          .span3
+            = level.properties['contained_level_names']
```

Comment:
nit; it'd look a lot nicer if you did something like `level.properties['contained_level_names'].join(", ")`